tools/run_tests/xds_k8s_test_driver/framework/xds_url_map_testcase.py

```diff
@@ -59,13 +60,21 @@
 RpcTypeUnaryCall = 'UNARY_CALL'
 RpcTypeEmptyCall = 'EMPTY_CALL'
 
+# All client languages
+ClientLangs = ('cpp', 'java', 'go', 'python')
```

Comment:
Nit: Shouldn't this global be IN_CAPS?

Reply:
Good point. Changed to `_CLIENT_LANGUAGES` and `_SERVER_LANGUAGES`.